docs/building-a-bento-amp-extension.md

```diff
@@ -92,130 +93,98 @@ The directory structure is below:
 
 In most cases you'll only create the required (req'd) files. If your element does not need custom CSS or JSS, you don't need to create those files.
 
-## Extend AMP.PreactBaseElement
-
-All Preact-based Bento AMP extensions extend `AMP.PreactBaseElement`. Bento AMP extensions differ from classic AMP extensions because they are self-managing and independent, and therefore usable in a wider range of contexts beyond AMP pages, while still being fully integrated with the AMP environment when used in a fully AMP document.
-
-Important: Bento-enabled AMP components built upon a Preact component must also be usable in insolation in React development contexts.
-
-The configurations which bridge the Preact implementation of the component and its custom element counterpart in an HTML or AMP document are explained in the [AMP/Preact Bridge](#amppreact-bridge) section, and the callbacks which handle AMP- and DOM- specific mutability traits are explained in the [PreactBaseElement Callbacks](#preactbaseelement-callbacks) section. All of these are also explained inline in the [PreactBaseElement](https://github.com/ampproject/amphtml/blob/main/src/preact/base-element.js) class.
-
-### Element and Component classes
-
-The following shows the overall structure of your element implementation
-file (`extensions/amp-my-element/1.0/amp-my-element.js`). See [Experiments](#experiments) to make sure your component is experimentally gated if necessary.
+### Versioning
 
-```js
-import {func1, func2} from '../../../src/module';
-import {BaseElement} from './base-element'; // Preact base element.
-import {CSS} from '../../../build/amp-my-element-1.0.css';
-// more ES2015-style import statements.
+AMP runtime is currently in v0 major version. Extensions versions are
+maintained separately. If your changes to your non-experimental
+extension makes breaking changes that are not backward compatible you
+should version your extension. This would usually be by creating a 0.2
```

Comment:
```suggestion
should release a new version of your extension. This would usually be by creating a [0.2 or 1.0](./spec/amp-versioning-policy.md#amp-extensions)
```